### DISCUSSION TITLE
Adding some customization on the tools to toggle the armour stand texture

### DISCUSSION THREAD

**Discussion by TimothyGrass** [SuperLlama88888/holoprint#16] [General]
For example, allow users to switch some of the blocks, such as planks to other frequently used tools, maybe sword. Aiming to enhance its user-friendliness

**Comment by SuperLlama88888** [comment]
I'm planning to add this soon, expect it by the end of November.

**Comment by SuperLlama88888** [comment]
This has been added in #24.
